### PR TITLE
Fix the bison flag check

### PR DESCRIPTION
**Pull request by gtrepta**
There's a `\n` escape sequence in an echo command in the Makefile, but echo needs `-e` passed in to properly evaluate it, and it's missing. This fixes that.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ FROM runtimeverificationinc/kframework-k:ubuntu-focal-${K_COMMIT}
 RUN    apt-get update            \
     && apt-get upgrade --yes     \
     && apt-get install --yes     \
+            bison                \
```
Comment (gtrepta) [inline Dockerfile:7]: The focal package for bison is only version 3.5.1, which will fail the version check.

In order to get CI to use the bison generated parser, you'll need to add a RUN instruction to the dockerfile which builds and installs a more recent version of bison.

**Comment by SchmErik** [comment]
@gtrepta I think I'm getting a little tunnel-visioned but I'm not sure why this is failing. I can't reproduce this locally and I'm using these steps to build/install bison... Any ideas??

**Comment by SchmErik** [comment]
Ah, I think I found the issue: the docker image from K installs bison using apt so it's outdated. The RUN command should probably go into the K repo's dockerfile.

**Comment by SchmErik** [comment]
This PR got a little messy. See #317 closing...

**Pull request closed without merging**